### PR TITLE
Fix cozero complemented error, add indiscrete theorems

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

File: theorems/T000252.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if:
+  P000129: true
+then:
+  P000015: true
+refs:
+  - mathse: 3844039
+    name: What topological properties are trivially/vacuously satisfied by any indiscrete space?
+---
+
+No disjoint closed sets exist.
```
Comment (prabau) [inline theorems/T000252.md:12]: "nonempty closed sets" ?

**Review by prabau — COMMENTED** [review]
Thanks for the fix.  In space S4 (indiscrete topology on two-point set), can we remove the asserted P16 since it will deduced from the new T251?  (Comparing the text in T251 and S4/P16 to see if one has a pedagogical advantage over the other, but it would be nice to be able to remove S4/P16)

**Comment by StevenClontz** [comment]
> Thanks for the fix. In space S4 (indiscrete topology on two-point set), can we remove the asserted P16 since it will deduced from the new T251? (Comparing the text in T251 and S4/P16 to see if one has a pedagogical advantage over the other, but it would be nice to be able to remove S4/P16)

implemented in c2dc802

**Comment by StevenClontz** [comment]
> > Thanks for the fix. In space S4 (indiscrete topology on two-point set), can we remove the asserted P16 since it will deduced from the new T251? (Comparing the text in T251 and S4/P16 to see if one has a pedagogical advantage over the other, but it would be nice to be able to remove S4/P16)
> 
> implemented in [c2dc802](https://github.com/pi-base/data/commit/c2dc80225dd5224892e29d74e596db36284ae303)

er, actually in 64c7e21

**Review by prabau — APPROVED** [review]

**Review by lyengulalp — APPROVED** [review]

**Pull request merged**